dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1688,12 +1703,131 @@
+        "type": "string",
+        "x-ms-enum": {
+          "name": "EntityCategory",
+          "modelAsString": true
+        },
+        "enum": [
+          "Default",
+          "All",
+          "Address",
+          "Numeric",
+          "Age",
+          "Currency",
+          "Number",
+          "NumberRange",
+          "Percentage",
+          "Ordinal",
+          "Temperature",
+          "Dimension",
+          "Length",
+          "Weight",
+          "Height",
+          "Speed",
+          "Area",
+          "Volume",
+          "Information",
+          "Temporal",
+          "Date",
+          "Time",
+          "DateTime",
+          "DateRange",
+          "TimeRange",
+          "DateTimeRange",
+          "Duration",
+          "SetTemporal",
+          "Event",
+          "SocialEvent",
+          "CulturalEvent",
+          "NaturalEvent",
+          "Location",
+          "GPE",
+          "City",
+          "Stage",
+          "CountryRegion",
+          "Continent",
+          "Structural",
+          "Geological",
+          "Organization",
+          "MedicalOrganization",
+          "StockExchange",
+          "SportsOrganization",
+          "Person",
+          "PersonType",
+          "Email",
+          "URL",
+          "IP",
+          "PhoneNumber",
+          "Product",
+          "ComputingProduct",
+          "Skill"
+        ]
+      },
+      "type": "array",
+      "uniqueItems": true
+    },
+    "BaseEntityOverlapPolicy": {
+      "description": "The abstract base class for entity OverlapPolicy.",
+      "type": "object",
+      "discriminator": "policyKind",
+      "properties": {
+        "policyKind": {
+          "description": "The entity OverlapPolicy object kind.",
+          "type": "string",
+          "default": "matchLongest",
+          "enum": [
+            "matchLongest",
+            "allowOverlap"
+          ],
+          "x-ms-enum": {
+            "name": "policyKind",
+            "modelAsString": true
+          }
+        }
+      },
+      "required": [
+        "policyKind"
+      ]
+    },
+    "MatchLongestEntityPolicyType": {
+      "description": "Represents the Match longest overlap policy.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseEntityOverlapPolicy"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "matchLongest"
+    },
+    "AllowOverlapEntityPolicyType": {
+      "description": "Represents the allow overlap policy.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseEntityOverlapPolicy"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "allowOverlap"
+    },
     "EntitiesTaskParameters": {
       "type": "object",
       "description": "Supported parameters for an Entity Recognition task.",
       "properties": {
         "stringIndexType": {
           "$ref": "common.json#/definitions/StringIndexType"
+        },
+        "inclusionList": {
```

Comment:
These should have descriptions, especially since, IIRC, you'll use the inclusion list then exclude as needed. IIRC, you also don't allow overlap. All those rules should be codified.